src/components/ContentSidebar/ContentSidebar.js

```diff
@@ -589,15 +589,30 @@ class ContentSidebar extends PureComponent<Props, State> {
         }
     }
 
+    /**
+     * Adds a comment to the comments state.
+     *
+     * @param {Comment} comment - The newly created comment from the API
+     * @return {void}
+     */
+    onCommentCreateSuccess(comment: Comment): void {
+        const { comments } = this.state;
+        if (comments && comments.entries) {
+            const newComments = { ...comments };
+            newComments.entries.push(comment);
+            this.setState({ comments: newComments });
```

Comment:
Do you want to increment the total_count as well? Also, this is a mutation, can do something similar to what you did above

Reply:
Good idea! Done